src/features/MainPage/index.js

```diff
@@ -82,14 +63,15 @@ function MainPage() {
           <Pagination
             currentPage={currentPage}
             minPageLimit={1}
-            maxPageLimit={100}
+            maxPageLimit={400}
+            url={toMainPage()}
           />
         </Container>
       ) : (
-        <ErrorPage />
```

Comment:
Why not? What we show customer if the data woun't be uploaded?

Reply:
I created this branch before the merge of the SearchResults, so then there was not an ErrorPage in our project. So like LoadingPage I left it to add later